merger

```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
+    # 1. Combine the two numbers into an average value
+    average_value = (textual_chance + numerical_chance) / 2
+
+    # 2. Display the median of the two values
```

Comment:
Output instead of display, display is for printing/GUIs, outputting/returning is for functions